### DISCUSSION TITLE
任务栏无显示应用

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4398] [Troubleshooting | 故障排查]
https://bbs.deepin.org/phone/zh/post/256641
任务栏无显示应用，具体步骤如下:
打开部分应用，任务栏无显示。

**Comment by BLumia** [comment]
这个信息完全不够呀。版本与复现步骤？

关于判断信息够不够，我给个简单的判断标准：对方提供的信息能让你自己复现问题吗？如果不能，那肯定是不够的。

**Reply by Linux-qitong** [reply]
这个是V23beta版本，就是打开部分应用任务栏不显示图标，而且我又没安装deepinV23😂

**Reply by BLumia** [reply]
按这个描述是严重问题，会阻塞发版的，所以发出去的版本不可能有这样的问题...

我这边没复现，但我没用 GIMP 来测试（开发源里 GIMP 的依赖有点问题，还待解决）。是只有 GIMP 能用来复现吗？

**Reply by Linux-qitong** [reply]
应该是

**Reply by BLumia** [reply]
找了个 v23 beta 非开发的环境测了下，也没有复现。尝试的步骤就是在 v23 beta 干净的环境下直接 `apt install gimp` 然后从 launcher 启动 GIMP。任务栏（dock）是有图标的。
